Submitted By: DJ Lucas <dj AT linuxfromscratch DOT org>
Date: 2006-06-25
Initial Package Version: 1.5.0_05
Upstream Status: Not applicable
Origin: Previous motif fixes patch
Description: The waiters demo has been broken for some time, since gcc-3.3.
             This patch keeps it from being compiled.

--- jdk-build-orig/j2se/make/mkdemo/jvmti/Makefile	2007-02-10 15:22:33.000000000 -0600
+++ jdk-build/j2se/make/mkdemo/jvmti/Makefile	2007-02-10 04:51:54.000000000 -0600
@@ -25,8 +25,7 @@
 	  heapViewer \
 	  heapTracker \
 	  minst \
-	  mtrace \
-	  waiters
+	  mtrace
 all build clean clobber::
 	$(SUBDIRS-loop)
 
